package.json

```diff
@@ -44,6 +45,9 @@
     "eslint": "7.32.0",
     "eslint-plugin-jsdoc": "^37.0.3",
     "eslint-plugin-react": "^7.26.1",
+    "eslint-plugin-import": "^2.25.2",
+    "eslint-plugin-unicorn": "21.0.0",
```

Comment:
Have to pin this, since [the upgrade is currently failing](https://github.com/sourcegraph/eslint-config/pull/227), so we should just keep parity for now.

Reply:
ok